packages/app/src/components/charts/StatisticsCard.tsx

```diff
@@ -8,13 +8,16 @@ interface GetIdentifierCountResponse {
   count: number
 }
 
+const apiUrl = process.env.REACT_APP_VERAMO_URL || 'http://localhost:3332/agent'
+const apiKey  = process.env.REACT_APP_VERAMO_API_KEY || 'test123'
```

Comment:
Can we place the local values into .env.example and add instruction in README?

Reply:
Also, should we move these to a config file?

Reply:
moved those to env file